tests/Flatbuffers.Test.Swift.WASM/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+import FlatBuffers_Test_Swift_WASMTests
+
+var tests = [XCTestCaseEntry]()
+tests += FlatBuffers_Test_Swift_WASMTests.__allTests()
+
+XCTMain(tests)
```

Comment:
There shouldn't be a need for `LinuxMain.swift` as `carton` uses test discovery by default.